modules/draco/src/index.js

```diff
@@ -9,13 +9,7 @@ export {default as DracoBuilder} from './draco-builder';
 // eslint-disable-next-line import/first
 import DracoParser from './draco-parser';
 
-const DEPRECATION_WARNING = `\
```

Comment:
+1.

Check all the worker loaders to see if there are any more of these warnings?